### PR TITLE
test: InstalledList tests

### PR DESCRIPTION
**Pull request by zoesteinkamp**
Closes #9639 IDPE

- [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass

### CODE REVIEW

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 50 | :red_circle:
accessibility | 65 | :yellow_circle:
best-practices | 87 | :green_circle:
seo | 80 | :yellow_circle:
pwa | null | :question:

File: src/templates/components/CommunityTemplatesInstalledList.test.tsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  )
+}
+
+describe('the Community Templates installed List', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  describe('handling loading the template list', () => {
+    it('the page loads in empty list', () => {
```
Comment (hoorayimhelping) [inline src/templates/components/CommunityTemplatesInstalledList.test.tsx:74]: could use some love

File: src/templates/components/CommunityTemplatesInstalledList.test.tsx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+      setup()
+      expect(screen.queryByText('Installed Templates')).not.toBeInTheDocument()
+      expect(
+        screen.queryByText(`You haven't installed any templates yet`)
+      ).toBeVisible()
+    })
+  })
+
+  describe('handle deleting template from template list', () => {
+    it('the delete is succesful', async () => {
```
Comment (hoorayimhelping) [inline src/templates/components/CommunityTemplatesInstalledList.test.tsx:84]: both descriptions can be made a little clearer

File: src/templates/components/CommunityTemplatesInstalledList.test.tsx [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+      const [notifyCallArguments] = mocked(notify).mock.calls
+      const [notifyMessage] = notifyCallArguments
+      expect(notifyMessage).toEqual(communityTemplateDeleteSucceeded('github'))
+
+      expect(
+        screen.queryByText(`You haven't installed any templates yet`)
+      ).toBeVisible()
+    })
+
+    it('the delete fails', async () => {
```
Comment (hoorayimhelping) [inline src/templates/components/CommunityTemplatesInstalledList.test.tsx:128]: >it the delete fails

think about how to make this sound a little more natural

**Review by hoorayimhelping — CHANGES_REQUESTED** [review]
great work on the tests! the descriptions could use a little bit of love though ;)

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**